testing_application/app.py

```diff
@@ -0,0 +1,82 @@
+from typing import TypedDict
+
+from robyn import Robyn, Request, jsonify, OpenAPI, SubRouter
+
+app = Robyn(
+    file_object=__file__,
+    openapi=OpenAPI(
+        openapi_title="Sample Pet Store App",
+        openapi_summary=" A pet store manager.",
+        openapi_description=" This is a sample server for a pet store.",
+        openapi_terms_of_service=" https://example.com/terms/",
+        openapi_version=" 1.0.1",
+        openapi_contact_name="API Support",
+        openapi_contact_url="https://www.example.com/support",
+        openapi_contact_email="support@example.com",
+        openapi_license_name="Apache 2.0",
+        openapi_license_url="https://www.apache.org/licenses/LICENSE-2.0.html",
+        # openapi_servers=[
+        #     {"url": "/", "description": "Debug environment"},
+        #     {"url": "https://example.com/api/v1/", "description": "Production environment"},
+        # ],
+        # openapi_external_docs={
+        #     "description": "Find more info here",
+        #     "url": "https://example.com/"
+        # }
```

Comment:
is this needed?

Reply:
These are another specs that the user may want to add. We're just supporting them.